src/templates/finops-hub/modules/hub.bicep

```diff
@@ -27,11 +27,14 @@ param tagsByResource object = {}
 @description('Optional. List of scope IDs to create exports for.')
 param exportScopes array
 
-@description('Optional. Number of days of cost data to retain in the ms-cm-exports container. Default: 0.')
-param exportRetentionInDays int = 0
+@description('Optional. Deploy Azure Data Explorer cluster for analytics. Default: false.')
+param deployDataExplorer bool = false
 
-@description('Optional. Number of months of cost data to retain in the ingestion container. Default: 13.')
-param ingestionRetentionInMonths int = 13
+// @description('Optional. Number of days of cost data to retain in the ms-cm-exports container. Default: 0.')
+// param exportRetentionInDays int = 0
+
+// @description('Optional. Number of months of cost data to retain in the ingestion container. Default: 13.')
+// param ingestionRetentionInMonths int = 13
```

Comment:
Why comment these out?

Reply:
Not being used, and cause a linter "error"